pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -21,14 +26,24 @@ const (
 type VPCNetworkConfigurationSpec struct {
 	// PolicyPath of Tier0 or Tier0 VRF gateway.
 	DefaultGatewayPath string `json:"defaultGatewayPath,omitempty"`
+
+	// VPCConnectivityProfile ID. This profile has configuration related to create VPC transit gateway attachment.
+	VPCConnectivityProfile string `json:"vpcConnectivityProfile,omitempty"`
+
+	// +kubebuilder:validation:Enum=SMALL;MEDIUM;LARGE;XLARGE
+	LbServiceSize string `json:"lbServiceSize,omitempty"`
+
 	// Edge cluster path on which the networking elements will be created.
 	EdgeClusterPath string `json:"edgeClusterPath,omitempty"`
```

Comment:
@lxiaopei : we should remove `DefaultGatewayPath`, `EdgeClusterPath`, `ExternalIPv4Blocks`, right?